Source/Scene/GroundPrimitive.js

```diff
@@ -807,6 +903,19 @@ define([
         }
     }
 
+    GroundPrimitive._initialized = false;
+    GroundPrimitive.init = function() {
+        var initialized = GroundPrimitive._initialized;
+        if (initialized) {
+            return when.resolve();
+        }
+
+        return loadJson(buildModuleUrl('Assets/approximateTerrainHeights.json')).then(function(json) {
```

Comment:
Ultimately, this data should probably come from the terrain server, but if we don't want to put that infrastructure in place, (i,e, have the GroundPrimitive ask the Terrain and the terrain implementations and fallback to this file if they don't have a custom implementation), then I'm okay with having it be GroundPrimitive specific for now (but I would like to hear what others think).

Also, I'm slightly concerned that this doesn't provide away for someone to specify their own heights, since it can produce incorrect viz if the terrain doesn't match, right?

Finally, we need to make sure the code that generated this is stored somewhere, how much code is it?